ui/widget.js

```diff
@@ -514,6 +514,16 @@ $.Widget.prototype = {
 			}
 		}
 
+		this._on( options.element, {
+			"remove": function( event ) {
+				$.each( that.classesElementLookup, function( key, value ) {
+					if ( $.inArray( event.target ) ) {
```

Comment:
Not sure about performance of `.not()` vs `inArray` but the line below also calls `.get` and then creates a new jQuery object which is what i was trying to avoid doing on every iteration since in many cases this object will contain many arrays most of which won't contain the element we are looking for.